plugins/outputs/http/http.go

```diff
@@ -168,8 +259,32 @@ func (h *HTTP) writeMetric(reqBody []byte) error {
 		req.SetBasicAuth(h.Username, h.Password)
 	}
 
-	req.Header.Set("User-Agent", internal.ProductToken())
-	req.Header.Set("Content-Type", defaultContentType)
+	// Authorization Code Grant (WIP)
```

Comment:
If this PR is still WIP, please mark it as draft and unmark when done. So we know when to do a final review..